requirements/docs.txt

```diff
@@ -5,7 +5,7 @@
 -r extras.txt
 -r install.txt
 docutils
-ipykernel
+ipykernel < 6.18.0
```

Comment:
```suggestion
ipykernel != 6.18.0
```
If this one release is the problem, don't we want to exclude it this way to allow 6.18.1?

Reply:
I got the same error with 6.18.1 as 6.18.0.  However, 6.18.2 was just released, so I'll try allowing that instead.